app/views/admin/consumer_apps/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="crayons-card p-3 mb-5 mt-2">
+  <%= link_to "New Consumer App", new_admin_consumer_app_path, class: "crayons-btn float-right" %>
+
+  <h1 class="crayons-title mb-6">Consumer Apps</h1>
+
+  <p>
+    The following are apps that will be able to receive Push Notifications from <%= SiteConfig.community_name %>.
+  </p>
+</div>
+
+<table class="crayons-table" width="100%">
+  <thead>
+    <tr>
+      <th scope="col">App Bundle</th>
+      <th scope="col">Platform</th>
+      <th scope="col">Devices</th>
+      <th scope="col">Authentication Key</th>
+      <th scope="col">Operational</th>
+      <th scope="col">Actions</th>
+    </tr>
+  </thead>
+  <tbody class="crayons-card">
+    <% @apps.each do |app| %>
+        <tr>
+          <td><%= app.app_bundle %></td>
+          <td><%= app.platform %></td>
+          <td><%= app.devices.count %></td>
+          <% if app.auth_credentials.present? %>
+            <td>*******</td>
+          <% else %>
+            <td>Not Available</td>
+          <% end %>
+          <% if app.operational? %>
+            <td><span class="crayons-icon" role="img" aria-label="Active">✅</span></td>
+          <% else %>
+            <td><span class="crayons-icon" role="img" aria-label="Active">❌</span></td>
```

Comment:
blocking: I think the label is incorrect here, as the UI indicates it's not active